gdc_client/upload/parser.py

```diff
@@ -88,8 +87,8 @@ def config(parser):
     parser.add_argument('--server', '-s',
                         default=defaults.tcp_url,
                         help='GDC API server address')
-    parser.add_argument('--part-size', '-ps',
-                        default=defaults.part_size,
+    parser.add_argument('--http-chunk-size', '-c',
```

Comment:
Make sure this is clearly documented in the ticket, so user services knows to relay this information to the users.

Reply:
got it